core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java

```diff
@@ -26,12 +28,19 @@
 
     @AttributeDefinition(name = "Get Cart Items Service Path", description = "This is the API path for getting cart item details", defaultValue = servicePathCartFetchItems, type = AttributeType.STRING )
     String cartFetch_servicePath_string() default servicePathCartFetchItems;
+    
+	@AttributeDefinition(name = "ADD_TO_CART_SERVICE_PATH", description = "This is add to cart API path", defaultValue = addToCartPath, type = AttributeType.STRING)
+	String magentoService_addToCartPath() default addToCartPath;
+	
+	@AttributeDefinition(name = "UPDATE_ITEM_QUANTITY_SERVICE_PATH", description = "This is update cart item API path", defaultValue = updateCartItemPath, type = AttributeType.STRING)
+	String magentoService_updateCartItemPath() default updateCartItemPath;
+
+	@AttributeDefinition(name = "SERVICE_PATH", description = "This is create empty cart API path", defaultValue = emptyCartPah, type = AttributeType.STRING)
+	String magentoService_emptyCartPath() default emptyCartPah;
 	
-	@AttributeDefinition(name = "GUEST CART ITEM DELETE SERVICE PATH", description = "This is guest cart item delete API path", defaultValue = guestCartItemDeletePah, type = AttributeType.STRING)
-    String deleteCartService_guestCartItemDeletePath() default guestCartItemDeletePah;
```

Comment:
@DeepaliAgrwl : this is used in /hcl/ecomm/core/services/impl/DeleteCartItemServiceImpl.java

Reply:
synced with deveploment now